packages/opentelemetry-plugin-dns/src/dns.ts

```diff
@@ -117,7 +117,6 @@ export class DnsPlugin extends BasePlugin<Dns> {
       plugin._logger.debug('wrap lookup callback function and starts span');
       const name = utils.getOperationName('lookup');
       const span = plugin._startDnsSpan(name, {
-        parent: plugin._tracer.getCurrentSpan(),
```

Comment:
ok but how dns span will be attached to the current ?

Reply:
parent span is now inferred from the parent context. When an incoming request comes in, a span is started and added to the context. When this `startSpan` call is run for the outgoing request, that span is automatically made the parent of this span.

Reply:
could you point me where this is done ? Thx

Reply:
https://github.com/dynatrace-oss-contrib/opentelemetry-js/blob/context-tracing/packages/opentelemetry-tracing/src/Tracer.ts#L68